site/docs/tasks/omd-discover-s3-asset.md

```diff
@@ -58,6 +58,6 @@ To create a connection to S3 and discovering your CSV asset:
    <br><center><img src="../../static/openmetadata/22_add_column_tag.jpg" width=600></center>
 1. Finally, press the `Custom Properties` tab
    <br><center><img src="../../static/openmetadata/23_select_custom_properties.jpg" width=600></center>
-1. Set the asset properties as needed. For instance, set `connectionType` to `s3`. If the discovered asset is a csv object, set `dataFormat` to `csv`
+1. Set the asset properties as needed. For instance, set `connectionType` to `s3`. If the discovered asset is a csv object, set `dataFormat` to `csv`. These values will be returned to the Fybrik Manager and will be instrumental in the construction of a data path
```

Comment:
Comment is true.  But didn't we say that these will be removed, and the values obtained automatically by the connector?

Reply:
@simanadler : There is a new PR which removes the `connectionType` custom property: https://github.com/fybrik/openmetadata-connector/pull/36
I don't see how we can remove the `dataFormat` property, but the PR also sets a default property for `dataFormat`. The default property is `csv`.
Once the PR is approved, I will update the documentation accordingly